float64[] t_value
float64 g_value
---
